### DISCUSSION TITLE
How to inherit a Theme from a Collection with the new site UI?

### DISCUSSION THREAD

**Discussion by a-asharapova** [GitbookIO/community#707] [0. General bugs and questions]
Hi guys! 

I cannot find the switch that enables Theme inheritance from a Collection to an individual space. I need to publish 6 spaces with the same 23 header links each.

Can you please tell me where the switch is now?

**Comment by addisonschultz** [comment]
Hi @a-asharapova! You will find all of the settings in the customization panel, if more than 1 space is published. You can read more about it here: https://docs.gitbook.com/published-documentation/customization/collection-customization

**Reply by a-asharapova** [reply]
Thank you @addisonschultz! Unfortunately, it does not solve my problem, as all spaces in a site must be variants of each other for this to work and mine are separate documents.
